examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/README.md

```diff
@@ -0,0 +1,356 @@
+write-firehose.iampolicy.awsblueprints.io             IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-s3.iampolicy.awsblueprints.io                   IAMPolicy         awsblueprints.io/v1alpha1   23m
+write-sqs.iampolicy.awsblueprints.io                  IAMPolicy         awsblueprints.io/v1alpha1   23m
+```
+
+### Build Lambda function packages
+
+This example uses ZIP packaging of the Lambda functions. 
+ - Navigate to `./src/logic/` directory
+ - Create ZIP file `microservice-business-logic.zip` with the `lambda_function.py` in it:
+    ```shell
+    zip microservice-business-logic.zip lambda_function.py
+    ```
+ - Navigate to `./src/authorizer/` directory
+ - Create ZIP file `microservice-authorizer.zip` with the `lambda_function.py` in it:
+    ```shell
+    zip microservice-authorizer.zip lambda_function.py
+    ```
+ - Create Amazon S3 bucket that will be used to store Lambda ZIP packages (make sure to use unique bucket name):
+   ```shell
+   aws s3api create-bucket --bucket <my-zip-packaging-bucket-name> --region <replace-with-aws-region> # example `us-east-1`
+   ```
+ - Upload both ZIP files to the S3 bucket you just created:
+   ```shell
+   aws s3 cp microservice-business-logic.zip s3://<my-zip-packaging-bucket-name>/ 
+   aws s3 cp microservice-authorizer.zip s3://<my-zip-packaging-bucket-name>/ 
+   ```
+
+### Update and apply the claim
+
+Make sure you are in the following directory:
+```shell
+cd examples/upbound-aws-provider/composite-resources/serverless-examples/microservice/
+```
+
+Set the AWS region in the claim with the ones used in the previous step “Build Lambda function packages” where the Lambda ZIP packages were uploaded to S3.
+```shell
+export AWS_REGION=<replace-with-aws-region> # example `us-east-1`
+export S3_BUCKET=<replace-with-s3-bucket-name> # example `my-crossplane-microservice-lambdas`
+```
+
+Change the default value for `CLAIM_NAME` with any name you choose.
+```shell
+export CLAIM_NAME=<replace-with-claim-name> # example `test-rest-api`
+```
+
+Run the below commands to generate random password to be used by a Lambda Authorizer and store it in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)
+```shell
+export AUTHORIZER_PASSWORD=$(aws secretsmanager get-random-password --output text)
+export SECRET_ARN=$(aws secretsmanager create-secret --name "$CLAIM_NAME-auth-password" --secret-string "$AUTHORIZER_PASSWORD" --output json | jq .ARN | tr -d '"')
+```
+
+Run the below command to use the template file `microservice-claim-tmpl.yaml` in the `claim` folder to create the claim file with the variables `CLAIM_NAME`, `S3_BUCKET`, and `AWS_REGION` substituted.
+```shell
+envsubst < "claim/microservice-claim-tmpl.yaml" > "claim/microservice-claim.yaml"
+```
+
+Note that you can retrieve password from the AWS Secrets Manager using following command:
+```shell
+ aws secretsmanager get-secret-value --secret-id "$CLAIM_NAME-auth-password"     
+```
```

Comment:
This should be moved 5 lines up before creating the claim from the template.

Reply:
Moved, added explanation on how it uses environmental variable and this command is used just if you need to retrieve password later